### PR TITLE
Improve readability of health scanners

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Makes health analyzers significantly more readable by removing some unnecessary information and moving it to a textbox.
Also, changes the colors for oxyloss to make it significantly more readable on darkmode.

## Why It's Good For The Game
Keeping up with a health analyzer with a busy chat can be a pain. This should make it easier to more quickly find the most important information from an analyzer usage in medbay.

## Images of changes
image
image

## Testing
Punched myself a few times, drank all the cryoxadone and sugar, then analyzed myself.

## Changelog
:cl:
tweak: Health analyzers should now be a fair bit more readable.
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -127,13 +136,14 @@ REAGENT SCANNER
...
-		to_chat(user, "\t Key: <font color='blue'>Suffocation</font>/<font color='green'>Toxin</font>/<font color='#FFA500'>Burns</font>/<font color='red'>Brute</font>")
-		to_chat(user, "\t Damage Specifics: <font color='blue'>?</font> - <font color='green'>?</font> - <font color='#FFA500'>?</font> - <font color='red'>?</font>")
-		to_chat(user, "<span class='notice'>Body Temperature: [M.bodytemperature-T0C]&deg;C ([M.bodytemperature*1.8-459.67]&deg;F)</span>")
-		to_chat(user, "<span class='warning'><b>Warning: Blood Level ERROR: --% --cl.</span><span class='notice'>Type: ERROR</span>")
-		to_chat(user, "<span class='notice'>Subject's pulse: <font color='red'>-- bpm.</font></span>")
-		return
+
+		msgs += "<span class='notice'>Analyzing Results for ERROR:\n\t Overall Status: ERROR</span>"
+		msgs += "\t Key: <font color='aqua'>Suffocation</font>/<font color='green'>Toxin</font>/<font color='#FFA500'>Burns</font>/<font color='red'>Brute</font>"
+		msgs += "\t Damage Specifics: <font color='aqua'>?</font> - <font color='green'>?</font> - <font color='#FFA500'>?</font> - <font color='red'>?</font>"
```
Comment (Contrabang) [inline code/game/objects/items/devices/scanners.dm:142]: Can you make this its own color in browser css? I cant read this color in lightmode
```suggestion
		msgs += "\t Damage Specifics: <font color='healthscan'>?</font> - <font color='green'>?</font> - <font color='#FFA500'>?</font> - <font color='red'>?</font>"
```

Comment (lewcc) [reply code/game/objects/items/devices/scanners.dm:142]: Yeah I can give that a go

**Comment by lewcc** [comment]
image
image

Thanks for the heads up, got some better colors now.

File: goon/browserassets/css/browserOutput.css [hunk 2]
```diff
@@ -379,6 +379,7 @@ h1.alert, h2.alert			{color: #000000;}
 .orangei				{color: #ffa500; font-style: italic;}
 .orangeb				{color: #ffa500; font-weight: bold;}
 .resonate				{color: #298F85;}
+.healthscan_oxy			{color: #007ECC;}
```
Comment (Contrabang) [inline goon/browserassets/css/browserOutput.css:382]: Something like would be better (heres the website btw https://webaim.org/resources/contrastchecker/)
```suggestion
.healthscan_oxy			{color: #0074BD;}
```
image

**Review by Contrabang — APPROVED** [review]
Code is technically fine, but I'd prefer if this was done eventually for the robot-scanner too, because a lot of their code is the same

**Review by S34NW — APPROVED** [review]

**Pull request merged**